setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
when run from inside `wizard_builder`, eg `pip install requirements/dev.txt`

dev points to `base.txt`, which does `-e .` which points to `setup.py` and grabs the dependencies from `install_requires`